src/GovUk.Education.ExploreEducationStatistics.Content.Services/MethodologyService.cs

```diff
@@ -22,29 +23,34 @@ public class MethodologyService : IMethodologyService
         private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
         private readonly IMapper _mapper;
         private readonly IMethodologyVersionRepository _methodologyVersionRepository;
+        private readonly IRedirectsCacheService _redirectsCacheService;
 
         public MethodologyService(ContentDbContext contentDbContext,
             IPersistenceHelper<ContentDbContext> persistenceHelper,
             IMapper mapper,
-            IMethodologyVersionRepository methodologyVersionRepository)
+            IMethodologyVersionRepository methodologyVersionRepository,
+            IRedirectsCacheService redirectsCacheService)
         {
             _contentDbContext = contentDbContext;
             _persistenceHelper = persistenceHelper;
             _mapper = mapper;
             _methodologyVersionRepository = methodologyVersionRepository;
+            _redirectsCacheService = redirectsCacheService;
         }
 
         public async Task<Either<ActionResult, MethodologyVersionViewModel>> GetLatestMethodologyBySlug(string slug)
         {
             return await _persistenceHelper
                 .CheckEntityExists<Methodology>(
                     query => query
-                        .Where(mp => mp.Slug == slug))
+                        .Include(m => m.LatestPublishedVersion)
+                        .Where(m =>
+                            m.LatestPublishedVersion != null
+                            && slug == (m.LatestPublishedVersion.AlternativeSlug ?? m.OwningPublicationSlug))) // slug == mv.Slug doesn't translate
```

Comment:
Same as per my other comment regarding pulling into memory.  Either that, or maybe transpose your other comment here?

Reply:
I left the other instances alone so I'll leave this one alone too - prioritising avoiding having to pull every latest published methodology version out of the database.